app/models/host/managed.rb

```diff
@@ -462,6 +462,16 @@ def host_params
     @cached_host_params = hp
   end
 
+  def host_inherited_params_objects
```

Comment:
Sure, but they're implementing exactly the same logic - that should be avoided and combined.